src/main/java/javaserver/Response.java

```diff
@@ -1,6 +1,7 @@
 package javaserver;
 
 public class Response {
+
 	private String statusLine;
 	private String header;
 	private String body;
```

Comment:
I'm personally not a huge fan of public setters since they allow the mutation of an object after its creation. It may be worth using the Builder pattern here.